content/post/21-10-01-progress-report-27.md

```diff
@@ -0,0 +1,254 @@
+- Physics multi-threading.
+- Some engineering items now play sounds when landing.
+- Throwing now feels more realistic as it comes to a sudden stop at the end of the throw.
+- Lerping has been fixed and pulling lerping has been implemented. In English: Movement feels *much* better.
+
+- Examine now shows what people have in their hands. *Contributed by T-Stalker*
+- Vox plushie.
+
+- Gas tanks can be inserted into canisters. *Contributed by Vera*
+- Smoking actually makes you inhale nicotine/THC reagents and you can snuff them out.
+- Dual-Port air vent.
+
+- Crematorium makes a sound during cremation and burns quicker. *Contributed by Swept*
+- Adds and updates a ton of inhands for items across the codebase.
+- Updated Botany tool sprites.
+- Announcements now play the announcement sound.
+- Admin ghosts can interact with stuff.
+- Centcom Official ghost role.
+- Jumpskirt for Captain.
+- Thrown bags of flours and tomatoes will now splat and spill their contents.
+
+## Technical
+
+### Section about OpenDream and extrapolate on Space Wizards involvement in the project.
+
+### Shuttles again
+metalgearsloth: How long do we want this to be? I would mainly just rant about having to deal with cross-broadphase shenanigans was CBT which was the only reason I didn't do it sooner (that and box2d made it way easier)
+
+### Section about PJB's power system.
+
+<!-- You could probably just cut some interesting parts from https://hackmd.io/@ss14/lowpower to write this part better. It could also go in content with a nice video. -->
+
+Rewrote most of the power system.
+
+All power supplying objects now have a built-in ramping system. If a large amount of extra power is suddenly needed, engines and batteries on the grid need a bit of time to "ramp up" to match.
+
+Power is passed through batteries immediately. This means that APCs are basically on standby until engine power or such cuts out. When the engine cuts out, heavy electrical grids will be underpowered for a second or two until backup batteries can spin up to fill the gap.
+
+## Patrons
+
+Thanks to Blaise M., Jack Rose, The Hateful Flesh, Prof. Omii, mksky, Alex Tempest, Rat Fungus, Will M., spinnermaster, Valinov, Dan Griskardian, S.C., "'-Y-", Jubal the Wizard, clyf, Kerb 755, Robin Rottstock, Altana, Durp, Joshington Awesomahee, Eric VW, Evan Armstrong, Enzoman12, Mono, Ethan Keller, DramaBuns, KrystalDisc, Darren Brady, Zandario, Anthony Fleck, vifs_vestige, Bobberunio, CPM311, ThatGuyGW, Farewell Fire, Await Future, Daniel Thompson, and Tomeno.
+
+## Credits
+
+The contributors since the last progress report were: 20kdc, Acruid, AJCM-git, boiled-water-tsar, clyfordv, ColdAutumnRain, DrSmugleaf, ElectroJr, Fiftyllama, Fogapod, GalacticChimp, ike709, Jaskanbe, Macoron, metalgearsloth, michaelcooke, mirrorcult, PaulRitter, PJB3005, plinyvic, scaly-chimp, scrato, SethLafuente, ShadowCommander, Silvertorch5, Swept, TimrodDX, Visne, Ygg01 and Vera.
```

Comment:
```suggestion
The contributors since the last progress report were: 20kdc, Acruid, AJCM-git, boiled-water-tsar, clyfordv, ColdAutumnRain, DrSmugleaf, ElectroJr, Fiftyllama, Fogapod, GalacticChimp, ike709, Jaskanbe, Macoron, metalgearsloth, michaelcooke, mirrorcult, PaulRitter, PJB3005, plinyvic, scaly-chimp, scrato, SethLafuente, ShadowCommander, Silvertorch5, Swept, TimrodDX, Visne, Ygg01 and Zumorica.
```